locales/base/translation.json

```diff
@@ -1860,6 +1860,41 @@
       "withdraw": "Transfer tokens to a bank account, mobile money, gift card etc."
     }
   },
+  "sendRedesign": {
```

Comment:
"sendRedesign" is funky naming because it only really makes sense right now, as we are updating the designs on the send flow. Within a few months this should be the only send flow that exists, making the "redesign" part quickly outdated, and if/when send designs are updated again, it will be actively confusing. I would even prefer "send2" or "sendV2", since at least we can increment the version number as designs evolve

This wouldn't be such a big deal if it was just on the translation file, but it appears to be all over the PR, from filenames to analytics events, components, screens, variable names, ...

Reply:
+1 to renaming this. maybe `SendSelectRecipient` is an option?

Reply:
Yeah -- I like `SendSelectRecipient`! Adjusted naming throughout.